configuration/pih/liquibase/sql/create_functions.sql

```diff
@@ -320,7 +320,79 @@ BEGIN
 
 END
 #
+/*
+patient birthdate
+*/
+#
+DROP FUNCTION IF EXISTS birthdate;
+#
+CREATE FUNCTION birthdate(
+    _patient_id int
+)
+	RETURNS date
+    DETERMINISTIC
 
+BEGIN
+    DECLARE  patientBirthdate date;
+
+	select birthdate into patientBirthdate from person where person_id = _patient_id and voided =0;
```

Comment:
@mogoodrich can you remove the voided check here please?

Reply:
Yep, already did it in a follow-on commit